### ISSUE TITLE
All losses should include required normalization

### ISSUE THREAD

**Issue by ivkireev86** [dllllb/pytorch-lifestream#79]
Contrastive - L2
BarlowTwins - BatchNorm(affine=False)
...
